Userland/Utilities/init.cpp

```diff
@@ -124,7 +124,7 @@ static ErrorOr<void> prepare_bare_minimum_devtmpfs_directory_structure()
     TRY(populate_device_node_with_symlink(DeviceNodeType::Character, "/dev/mem"sv, 0600, 1, 1));
     TRY(populate_device_node_with_symlink(DeviceNodeType::Character, "/dev/null"sv, 0666, 1, 3));
     TRY(populate_device_node_with_symlink(DeviceNodeType::Character, "/dev/full"sv, 0666, 1, 7));
-    TRY(populate_device_node_with_symlink(DeviceNodeType::Character, "/dev/random"sv, 0666, 1, 8));
+    TRY(populate_device_node_with_symlink(DeviceNodeType::Character, "/dev/random"sv, 0644, 1, 8));
```

Comment:
Is there even such a thing as bad entropy? As far as I know/heard, you can't actually hurt the randomness of the RNG by mixing in bad data as long as it is mixed in properly.

Reply:
RHEL and FreeBSD restrict writes to /dev/random for the sake of hardening so we might adopt this as well